src/EditorFeatures/Test2/ReferenceHighlighting/AbstractReferenceHighlightingTests.vb

```diff
@@ -19,16 +20,16 @@ Imports Roslyn.Utilities
 Namespace Microsoft.CodeAnalysis.Editor.UnitTests.ReferenceHighlighting
     <[UseExportProvider]>
     Public MustInherit Class AbstractReferenceHighlightingTests
-        Protected Async Function VerifyHighlightsAsync(test As XElement, Optional optionIsEnabled As Boolean = True) As Tasks.Task
-            Await VerifyHighlightsAsync(test, optionIsEnabled, outOfProcess:=False)
-            Await VerifyHighlightsAsync(test, optionIsEnabled, outOfProcess:=True)
+        Protected Async Function VerifyHighlightsAsync(test As XElement, Optional optionIsEnabled As Boolean = True) As Task
+            Await VerifyHighlightsAsync(test, optionIsEnabled, TestHost.InProcess)
```

Comment:
Consider looping through all values for the enum, so it doesn't need changes if we add a new member to the enum in future. Same change for other such cases in this PR.

Reply:
We can do that later on if desired. I'd rather not add to this PR.